k8s/aat/common/sscs/sscs-case-loader.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: sscs-case-loader
+  namespace: sscs
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: sscs-case-loader
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: sscs-case-loader
+    version: 0.0.1
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "e2300a5f-a16f-4657-a63b-41913805661e"
```

Comment:
You shouldn't need this as its already here : https://github.com/hmcts/sscs-case-loader/blob/master/src/main/resources/bootstrap.yaml#L24